wdl-engine/src/backend.rs

```diff
@@ -294,13 +284,21 @@ pub trait TaskExecutionBackend: Send + Sync {
         hints: &HashMap<String, Value>,
     ) -> Result<TaskExecutionConstraints>;
 
-    /// Gets the container root directory for the backend (e.g. `/mnt/task`)
+    /// Gets the container (guest) root directory for the backend (e.g.
+    /// `/mnt/task`).
+    ///
+    /// Containerized execution uses the following guest paths:
+    ///
+    /// * `<root>/inputs` - where inputs are mounted; this is mounted read-only.
+    /// * `<root>/work` - the task working directory; this is mounted
+    ///   read-write.
+    /// * `<root>/command` - the command to execute; this is mounted read-only.
```

Comment:
```suggestion
    /// * `<root>/inputs/` - where inputs are mounted; this is mounted read-only.
    /// * `<root>/work/` - the task working directory; this is mounted
    ///   read-write.
    /// * `<root>/command` - the command to execute; this is mounted read-only.
```
nit to clarify that the first two are directories and the last is a file